deployment/keploy/README.md

```diff
@@ -15,4 +15,5 @@ kubectl --namespace default port-forward $POD_NAME 8080:$CONTAINER_PORT
 Then the keploy service should be accessible on http://127.0.0.1:8080
 
 ## Access via ingress
-To access Keploy though ingress, please add information about ingress in the [values.yaml](values.yaml) file. 
\ No newline at end of file
+To access Keploy though ingress, please add information about ingress in the [values.yaml](values.yaml) file. 
+To Set prefix path update env value KEPLOY_PATH_PREFIX in [values.yaml], and PATH_PREFIX in docker file to build UI image with custom prefix path.
```

Comment:
To host keploy on a subpath of a domain instead of a subdomain (eg: example.com/keploy). You can set the env value KEPLOY_PATH_PREFIX in [values.yaml] and then build a custom docker image with the `KEPLOY_PATH_PREFIX` argument set. Then you have to use this docker image in your helm chart.